Mathlib/AlgebraicGeometry/AffineSpace.lean

```diff
@@ -0,0 +1,364 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Algebra.MvPolynomial.Monad
+import Mathlib.AlgebraicGeometry.Limits
+
+/-!
+# Affine space
+
+## Main definitions
+- `AlgebraicGeometry.AffineSpace`: `𝔸(n; S)` is the affine `n`-space over `S`.
+- `AlgebraicGeometry.AffineSpace.coord`: The standard coordinate functions on the affine space.
+- `AlgebraicGeometry.AffineSpace.homOfVector`:
+  The morphism `X ⟶ 𝔸(n; S)` given by a `X ⟶ S` and a choice of `n`-coordinate functions.
+- `AlgebraicGeometry.AffineSpace.homOverEquiv`:
+  `S`-morphisms into `Spec 𝔸(n; S)` are equivalent to the choice of `n` global sections.
+- `AlgebraicGeometry.AffineSpace.SpecIso`: `𝔸(n; Spec R) ≅ Spec R[n]`
+
+-/
+
+open CategoryTheory Limits MvPolynomial
+
+noncomputable section
+
+namespace AlgebraicGeometry
+
+universe u
+
+variable (n : Type) (S : Scheme.{u})
```

Comment:
The PR looks good to me. However, would there be any significant issue if this started as follows:
```suggestion
variable (n : Type u) (S : Scheme.{max u v})
```

Reply:
Seems like it just works. Thanks!